core/deployment/src/main/java/io/quarkus/deployment/dev/RuntimeUpdatesProcessor.java

```diff
@@ -286,6 +287,13 @@ public DevModeType getDevModeType() {
 
     @Override
     public boolean doScan(boolean userInitiated) throws IOException {
+        return doScan(userInitiated, false);
+    }
+
+    public boolean doScan(boolean userInitiated, boolean force) throws IOException {
```

Comment:
I assume you plan to use `force` in the future

Reply:
It's used in the line below. It needs to be true for the user to force a scan when it is disabled.